### PR TITLE
added Glowdust Layer feature to Glowstone canyon biome

### PR DESCRIPTION
**Pull request by TelepathicGrunt**
title says it all

### CODE REVIEW

File: src/main/java/com/nekomaster1000/infernalexp/init/IEFeatures.java [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
 import com.nekomaster1000.infernalexp.InfernalExpansion;
-import com.nekomaster1000.infernalexp.world.gen.features.BoulderFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.DullthornsFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.GlowSpikeFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.GlowstoneRavineFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.HangingGiantBrownMushroomFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.LuminousFungusFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.ShroomlightTearFeature;
-import com.nekomaster1000.infernalexp.world.gen.features.SinkHoleFeature;
+import com.nekomaster1000.infernalexp.world.gen.features.*;
```
Comment (Hellion-Games) [inline src/main/java/com/nekomaster1000/infernalexp/init/IEFeatures.java]: Swan is gonna freak about this one

Comment (Hellion-Games) [inline src/main/java/com/nekomaster1000/infernalexp/init/IEFeatures.java]: Don't use Wildcards -Swan Probably

File: src/main/java/com/nekomaster1000/infernalexp/world/gen/features/GlowLayerFeature.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        }
+        return false;
+    }
+
+    private int numberOfGlowdustSandNearby(ISeedReader world, BlockPos.Mutable mutableBlockPos, BlockPos.Mutable mutableBlockPosNeighbors) {
+        int glowdustSandCount = 0;
+        int radius = 2;
+        for (int x = -radius; x <= radius; x++) {
+            for (int z = -radius; z <= radius; z++) {
+                if(x == 0 && z == 0) continue;
```
Comment (Hellion-Games) [inline src/main/java/com/nekomaster1000/infernalexp/world/gen/features/GlowLayerFeature.java]:
```suggestion
                if (x == 0 && z == 0) continue;
```

**Review by Dariensg — CHANGES_REQUESTED** [review]

File: src/main/java/com/nekomaster1000/infernalexp/init/IEFeatures.java (hunk 1 above)
Comment (SwanX1) [reply src/main/java/com/nekomaster1000/infernalexp/init/IEFeatures.java]: definitely*

**Review by Dariensg — APPROVED** [review]

**Comment by Dariensg** [comment]
LGTM

**Pull request merged**